docs/docs/styled-components.md

```diff
@@ -112,11 +112,11 @@ export default () => (
 )
 ```
 
-### Enabling user stylesheets with a stable class name
+### Ativando folhas de estilo do usuário com um nome de classe estável
 
-Adding a persistent CSS `className` to your styled components can make it easier for end users of your website to take advantage of [user stylesheets](https://www.viget.com/articles/inline-styles-user-style-sheets-and-accessibility/) for accessibility.
+Adicionar um nome de classe CSS persistente aos seus componentes de estilo pode facilitar para os usuários finais do seu site para tirar proveito das [folhas de estilos do usuário](https://www.viget.com/articles/inline-styles-user-style-sheets-and-accessibility/) para acessibilidade.
```

Comment:
Acredito que deve-se manter como `className`  mesmo, pois faz referência ao `className` do react, porém na tradução também fica ok. Vou abrir esta Review para discutir com todos a melhor tradução. :)

Reply:
Ok, sem problemas :) mediante o que decidirem faço a alteração.

Reply:
Neste caso parece que ficou bem colocado, porque indica o que o CSS-in-JS faz. Talvez tentar trazer ambos para acostumar o desenvolvedor com o nome? Algo como:
`...nome de classe CSS (className) persistente aos seus componentes...`

Reply:
Acho uma boa deixar entre parênteses mesmo @Italox 🙂

E @junagao após esta alteração, está aprovado por mim! Obrigado pela contribuição 🎉